Entities/MovableObject.h

```diff
@@ -1493,6 +1493,28 @@ enum MOType
 
 	void SetImpulseOffset(int n, Vector v) { if (n > 0 && n < m_ImpulseForces.size()) m_ImpulseForces[n].second = v; }
 
+//////////////////////////////////////////////////////////////////////////////////////////
+// Virtual method:  GetUpdatePeriod()
```

Comment:
Please add lua bindings for this getter and setter. It's not a big deal I guess, but people may want to be able to change this on the fly and there's no reason not to support it.
If you don't use unsigned int (I'm not actually sure if luabind plays well with unsigned ints) you'll also need to put a safety check in the setter to be sure it's >= 0 (or maybe just > 0 since 0 and 1 are effectively the same)